src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/ElectronicServiceChannelInTestsIT.java

```diff
@@ -0,0 +1,152 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+
+import fi.metatavu.kuntaapi.server.rest.model.ElectronicServiceChannel;
+import fi.metatavu.ptv.client.model.V6VmOpenApiElectronicChannelInBase;
+import fi.metatavu.ptv.client.model.V7VmOpenApiElectronicChannel;
+import fi.otavanopisto.kuntaapi.server.persistence.model.clients.AccessType;
+import fi.otavanopisto.kuntaapi.server.persistence.model.clients.ClientOrganizationPermission;
+import fi.otavanopisto.kuntaapi.test.AbstractPtvMocker;
+
+public class ElectronicServiceChannelInTestsIT extends AbstractPtvInTest {
+
+  private static final String SERVICE_CHANNEL_FIND_PATH = "/electronicServiceChannels/{kuntaApiChannelId}";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1075)